bin/docker-build-cli-bin

```diff
@@ -5,42 +5,28 @@ set -eu
 . bin/_docker.sh
 . bin/_tag.sh
 
-if [ $# -eq 0 ]; then
-    tag="$(clean_head_root_tag)"
-elif [ $# -eq 1 ]; then
-    tag="${1:-}"
-else
-    echo "usage: $(basename $0) [tag]" >&2
-    exit 64
-fi
-
 dockerfile=cli/Dockerfile-bin
 
 validate_go_deps_tag $dockerfile
 
-(
-    unset DOCKER_FORCE_BUILD
-    bin/docker-build-base
-    bin/docker-build-go-deps
-) >/dev/null
+bin/docker-build-base 1>&2
+bin/docker-build-go-deps 1>&2
 
-IMG=$(docker_maybe_build . \
-    "$(docker_repo cli-bin)" \
-    "${tag}" \
-    $dockerfile)
+IMG=$(docker_build cli-bin "$(head_root_tag)" $dockerfile)
 
 ID=$(docker create "$IMG")
 
-for OS in linux macos ; do
+# copy the newly built conduit cli binaries to the local system
+for OS in darwin linux windows ; do
     DIR="target/cli/${OS}"
     mkdir -p "$DIR"
 
     if docker cp "$ID:/out/conduit-${OS}" "$DIR/conduit" ; then
         echo "$DIR/conduit"
     else
-        docker rm "$ID" >/dev/null
+        docker rm "$ID" 1>&2
```

Comment:
why?

Reply:
in general i prefer to see all the output docker is providing to let me know things are not hung, but i'll roll this back.